requirements/tests.txt

```diff
@@ -1,5 +1,4 @@
 # tests
-mypy==1.9.0
```

Comment:
I moved "mypy" under "style" in CI but forgot to move it in the requirements files.